packages/frontend/amp/components/AMPJSON.tsx

```diff
@@ -0,0 +1,12 @@
+// tslint:disable:react-no-dangerous-html
```

Comment:
If you can, disabling the specific line is better, but I realise it is not always easy to figure out how to apply it.

Reply:
👍